force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +524,14 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        // Compare the ages of the two persons
+        if (person1.getAge() > person2.getAge()) {
```

Comment:
Rather than have three separate conditional blocks, you can reduce to two by updating the first if to `person1.getAge() >= person2.getAge()` and then putting the alternative in an else